raiwidgets/tests/responsibleai_dashboard/test_responsibleai_dashboard_input_erroranalysis.py

```diff
@@ -70,6 +71,23 @@ def test_rai_dashboard_input_adult_matrix_failure(
             flask_server_prediction_output,
             "Failed to generate json matrix representation,")
 
+        # change metric back to be valid
+        metric = "Error rate"
+        # test invalid features throw errors
+        invalid_features = ['Age', 'InvalidFeature']
+        post_data = [invalid_features, filters, composite_filters,
+                     quantile_binning, num_bins, metric]
+        dashboard_input = ResponsibleAIDashboardInput(ri)
+
+        flask_server_prediction_output = dashboard_input.matrix(post_data)
+
+        self.check_failure_criteria(
+            flask_server_prediction_output,
+            "Failed to generate json matrix representation,")
+        self.check_failure_criteria(
+            flask_server_prediction_output,
+            "not found in dataset. Existing features")
```

Comment:
Consider adding another test case for this scenario. Overloading tests with more code make it difficult to understand and update the test.

Reply:
done!